com.github.thezbyg.gpick.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: com.github.thezbyg.gpick
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: gpick
+rename-icon: gpick
+rename-desktop-file: gpick.desktop
+rename-appdata-file: gpick.appdata.xml
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+modules:
+  - shared-modules/gtk2/gtk2.json
+  - shared-modules/lua5.3/lua-5.3.2.json
+  - boost.yaml
+  - adwaita-icon-theme.yaml
```

Comment:
Not to derail discussion above, it's unclear to me why does it need to be bundled. gpick just assumes Adwaita presence?

Reply:
Ah, you are not using GNOME runtime. Never mind.